From caaf35d652f5b187077d6f9eefc8fca287e934a5 Mon Sep 17 00:00:00 2001
From: Suneel Garapati <sgarapati@marvell.com>
Date: Thu, 19 Dec 2019 18:07:14 -0800
Subject: [PATCH 1044/1239] Revert "Revert "Fix PCIe console""

This reverts commit fe1b9119124c8123ee7673fae1e4f4dafd850559.
Now that pcie console shutdown is fixed, re-enable probe.

Change-Id: Id8cf026956c7d15a5cad3e46e617a75ecba2789c
Signed-off-by: Suneel Garapati <sgarapati@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/20611
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Suneel Garapati <sgarapati@caviumnetworks.com>
---
 board/Marvell/octeontx2/board.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/Marvell/octeontx2/board.c b/board/Marvell/octeontx2/board.c
index c15e970479..9e34860f38 100644
--- a/board/Marvell/octeontx2/board.c
+++ b/board/Marvell/octeontx2/board.c
@@ -23,7 +23,7 @@
 DECLARE_GLOBAL_DATA_PTR;
 
 #define BOOTCMD_NAME	"pci-bootcmd"
-#define CONSOLE_NAME	"pci-console"
+#define CONSOLE_NAME	"pci-console@0"
 
 extern unsigned long fdt_base_addr;
 extern void cgx_intf_shutdown(void);
-- 
2.29.0

